frontend/src/app/commonComponents/UserDetails/SpecialStatusNotice.tsx

```diff
@@ -0,0 +1,105 @@
+import React, { ReactNode, useState } from "react";
+
+import { SettingsUser } from "../../Settings/Users/ManageUsersContainer";
+import { OktaUserStatus } from "../../utils/user";
+import Button from "../Button/Button";
+
+import ReactivateUserModal from "./ReactivateUserModal";
+import ResendActivationEmailModal from "./ResendActivationEmailModal";
+import UndeleteUserModal from "./UndeleteUserModal";
+
+export const SpecialStatusNotice: React.FC<{
+  user: SettingsUser;
+  isUpdating: boolean;
+  onResendUserActivationEmail: (userId: string) => void;
+  onReactivateUser: (userId: string) => void;
+  onUndeleteUser: () => void;
+}> = ({
+  user,
+  isUpdating,
+  onResendUserActivationEmail,
+  onReactivateUser,
+  onUndeleteUser,
+}) => {
+  /**
+   * Setup labels and modal per status
+   */
+  let confirmationModal: JSX.Element = <></>;
+  let statusDescription: null | ReactNode = null,
+    handleStatusBtnLabel: string = "";
+
+  const [showConfirmationModal, setShowConfirmationModal] = useState(false);
+
+  const closeModal = () => setShowConfirmationModal(false);
+  if (user.status === OktaUserStatus.UPDATING) {
+    statusDescription =
+      "Account is being restored. Please allow a few moments for the user's record to update.";
```

Comment:
At least to me, this kind of feels like it will load the updated user's record automatically, but it doesn't. If I'm the only one I'm ok with leaving it as is.

Reply:
I felt that way too. Maybe we can encourage them to refresh the page? 😆 🤔

Reply:
What about, `Account is being restored. Refresh the page after a few moments to see the updated user record.`?

Reply:
It works for me! LMK @BobanL @emyl3 @jayna-SkylightDigital if that sounds good and I will push the update :D